lib/puppet/pops/parser/lexer2.rb

```diff
@@ -640,7 +640,7 @@ def file
   #
   def lex_file(file)
     initvars
-    contents = Puppet::FileSystem.exist?(file) ? Puppet::FileSystem.read(file) : ''
+    contents = Puppet::FileSystem.exist?(file) ? Puppet::FileSystem.read(file, :encoding => 'utf-8') : ''
```

Comment:
This method tells ruby to interpret the bytes in the file as `UTF-8` regardless of the default external encoding, right? Essentially, read the bytes, don't do any transcoding, and tag the string as UTF-8. If so, :+1: